src/server/pipeline/flow_script.py

```diff
@@ -129,6 +132,62 @@ def compare_names(n1, n2):
     return name_to_array(n1).bool_op("&&")(name_to_array(n2))
 
 
+def filter_invalid_pdp_data(conn):
```

Comment:
We could format this as one big statement, but I liked it separated. 

Strictly a styling consideration... I originally called each one `condition_#` but realized if we remove one we'd have to do some number-reordering. I don't love all of these names, but they work.